### DISCUSSION TITLE
We are unable to complete your signup. Please read this guide for further details Error

### DISCUSSION THREAD

**Discussion by maaltv** [vercel/community#5806] [Account Issue]
### Question

Hi while doing sign up i am getting this error "We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details." i tried to login few times system said your email already attached with giithub account use that email ,when i use that email then again "We are unable to complete your signup"

after trying multiple times i found support mail and sent email  to registration@vercel.com for registration related  help .it said Our typical review time is 1-2 weeks.

May i know any solution for this?
Thanks

**Comment by kshzz24** [comment]
I am also facing the same issue, when tried to login using GitHub account, it shows "Your account requires further verification. Please contact registration@vercel.com to complete your registration", same the when using Email We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details

**Comment by Mihai-github** [comment]
Also received the same error, did anyone found out the issue?

**Comment by IgorRSSantos** [comment]
Alguem conseguiu resolver esse problema?

**Comment by sabirali12** [comment]
I am also facing the same issue, when tried to login using GitHub account, it shows "Your account requires further verification. Please contact [registration@vercel.com](mailto:registration@vercel.com) to complete your registration", same the when using Email We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details

**Comment by ghost** [comment]
I am also facing the same issue, when tried to login using GitHub account, it shows "Your account requires further verification. Please contact [registration@vercel.com](mailto:registration@vercel.com) to complete your registration", same the when using Email We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details

**Comment by Akighir-Aondohemba** [comment]
I'm facing the same issues.

**Comment by AlejandroIrizarry** [comment]
Found the answer:

_`1. Set your main GitHub email address to Gmail`_
_`2. Sign up for Vercel using the email account`_
_`3. Import the project on GitHub`_

**Reply by mmolalekan** [reply]
Worked perfectly. Thanks a bunch

**Reply by AGEN233** [reply]
this not used for me，i ended up send email to vecerl  to resolve it

**Comment by AGEN233** [comment]
I am also facing the same issue, when tried to login using GitHub account, it shows "Your account requires further verification. Please contact [registration@vercel.com](mailto:registration@vercel.com) to complete your registration", same the when using Email We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details

**Comment by userName194** [comment]
Same problem, i'm trying to pass verification in order to deploy the site. I have an account on GitHub and am trying to get verified using it. However, I keep getting this message: "Your account requires further verification. Please contact registration@vercel.com to complete your registration." I go to the address "[registration@vercel.com](mailto:registration@vercel.com)", select "Sign Up", choose "Hobby", enter a name, and select "Continue with GitHub", but I keep getting the same error. Next, I select "Continue with Email", and get an error: "The email address already exists. Log in?". Next, I select "Log in" and choose "Continue with GitHub", and get the same error: "Your account requires further verification. Please contact registration@vercel.com to complete your registration." Next, i select "Continue with Email" from "Log in", and get an error: "You are enable to complete your signup. Please, read the guide fot further details."
I tried setting the browser to allow all cookies, changing browsers, turning off the VPN, created a special token on GitHub. I also contacted Vercel support and attached screenshots. The problem is still not solved.

**Reply by userName194** [reply]
The issue has been resolved. I sent a message to "registration@vercel.com" with the subject "Authorization and Registration Error". In the email, I described the login problem. About an hour later, I received a message in my inbox:

"Thank you for contacting Vercel! Your requested review for Vercel Registration has been opened and your case number is XXXXXXXX.

If you have any additional details to share, just hit "reply" to this email. Our team will see your follow-up.

Our typical review time is 1 week. We appreciate your patience during that time. Your case will be addressed in the order it was received, so duplicate submissions may negatively impact your position in the queue.

If you are unable to log in with a new account, your account may have been flagged for additional review during the registration process. To protect against platform abuse, Vercel employs multiple layers of security and verification during the account signup process, and depending on our investigation, we may be unable to approve your account or discuss the review further.

If you are seeking help with SMS verification and you did not include the phone number you are using when this request was submitted, please respond to this email with your phone number, including country code. We will need this information to help further. If you have been unable to receive the SMS, you will need to contact your phone carrier for additional help.

If you have an existing Vercel account and cannot access it, please provide us with as many details as possible about your reason for using Vercel, any error message you received while attempting to log in, your Git provider username, and other email addresses that may be associated.

Our team will reach out to you with an update from registration@vercel.com."

At the end of the email, there was a button "Attach Phone Number". When clicked, an additional message window opened where I entered:
"This is my number: 012345678912
case number is 01234567.
thread::codeIndicatedAtTheBottomOfThePage::"

About three hours later, I received a second response from Vercel:
"Hello,
Thank you for contacting our team for help with your Vercel registration. We appreciate your patience as our team ran checks that help ensure new users are not abusive or bots. We’ve approved your account for use on Vercel. We recommend looking at our Terms of Service and Fair Use Policy to understand what is allowed on our platform.
Please let us know if you have any questions while getting set up with Vercel!"

After that, access was granted.

[a comment by an automated account is not shown]
